File: 0001-two-sum/0001-two-sum.js
/**
 * @param {number[]} nums
 * @param {number} target
 * @return {number[]}
 */
var twoSum = function(nums, target) {
    let sum=[0,0];
    for(let i=0;i<nums.length;i++){
        for(let ii=i+1;ii<nums.length;ii++)
        if(nums[i]+nums[ii]==target){
            sum=[i,ii]
        }
    }
    return sum
};

File: 0001-two-sum/0001-two-sum.cs
public class Solution {
    public int[] TwoSum(int[] nums, int target) {
        int[] sum={0,0};
        for(int i=0;i<nums.Length;i++){
            for(int ii=i+1;ii<nums.Length;ii++){
                if(nums[i]+nums[ii]==target){
                    sum[0]=i;
                    sum[1]=ii;
                    return sum;
                }
            }
        }
        return nums;
    }
}